packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import { IntlProvider } from "react-intl";
+import { boolean, select, withKnobs } from "@storybook/addon-knobs";
```

Comment:
The [@storybook/addon-knobs](https://www.npmjs.com/package/@storybook/addon-knobs) package is deprecated. Please use [Storybook args and argTypes](https://storybook.js.org/docs/react/writing-stories/args) instead.

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/281/commits/da0eb14bbfb78531b95e966f84d82529b0c53db5.